CHANGELOG.md

```diff
@@ -1,3 +1,75 @@
+## v0.6.1 - 2021-02-09
+
+This is a patch release which includes mainly bug fixes.
+
+> **NOTE**: Please read the upgrading guidelines [here](https://github.com/kinvolk/lokomotive/blob/v0.6.1/CHANGELOG.md#upgrading-from-v060).
+
+### Changes in v0.6.1
+
+#### Development
+
+- Velero: Add tolerations to Restic plugin ([#1348](https://github.com/kinvolk/lokomotive/pull/1348)).
+- Velero: Add e2e tests ([#1353](https://github.com/kinvolk/lokomotive/pull/1353)).
+- Update all Go dependencies ([#1358](https://github.com/kinvolk/lokomotive/pull/1358)).
+-
+#### Terraform Provider Updates
+- Update Packet[Equinux Metal] terraform provider to 3.2.1 ([#1349](https://github.com/kinvolk/lokomotive/pull/1349)).
+
+#### Bug fixes
+
+- Prefix `ETCD_` for standard etcd environment variables only ([#1308](https://github.com/kinvolk/lokomotive/pull/1308)).
+- Update Restic TolerationSeconds type to integer and add conditional checks ([#1365](https://github.com/kinvolk/lokomotive/pull/1365)).
```

Comment:
This is more of a feature I think than a bug fix.

Reply:
Adding Tolerations to Velero is a feature, this to me felt like bug fix.

Reply:
Yeah, it's not a new feature, it's a fix for broken feature.